tests/test_absolute_heat.py

```diff
@@ -10,8 +10,13 @@ class TestAbsoluteHeat(TestCase):
     def test_absolute_heat(self):
         """
         This is a single line ring model, meaning that there are no dedicated supply or return
-        lines. This means that this test checks whether this model is correctly parsed from esdl
-        and meets the logical energy and heat to discharge constraints.
+        lines. This means that this model pipes are not related.
```

Comment:
I do not understand this sentence: "This means that this model pipes are not related.I"

Reply:
no relation between hot and cold pipes exists